website/docs/reference/node-selection/yaml-selectors.md

```diff
@@ -0,0 +1,181 @@
+---
+title: "YAML Selectors"
+---
+
+<Changelog>New in v0.18.0</Changelog>
+
+:::info [β] Beta Feature
+This is net-new functionality in v0.18.0, with iterative improvements to come.
+If you encounter unexpected behavior, please post in Slack or open an issue.
+:::
+
+Write model selectors in YAML, save them with a human-friendly name, and reference them using the `--selector` flag.
+By recording selectors in a top-level `selectors.yml` file:
+
+* **Legibility:** complex selection criteria are composed of dictionaries and arrays
+* **Version control:** selector definitions are stored in the same git repository as the dbt project
+* **Reusability:** selectors can be referenced in multiple job definitions, and their definitions are extensible (via YAML anchors)
+
+Selectors live in a top-level file named `selectors.yml`. Each has a `name` and a `definition`:
+
+<File name='selectors.yml'>
+
+```yml
+selectors:
+  - name: nodes_to_joy
+    definition: ...
+  - name: nodes_to_a_grecian_urn
```

Comment:
[community.getdbt.com/how-to-write-high-quality-documentation#using-obscure-greek-references](http://www.reactiongifs.com/r/jonw.gif)

Reply:
[all you need to know](https://www.poetryfoundation.org/poems/44477/ode-on-a-grecian-urn)